contrib/automation_tests/fragments/move_tabs.py

```diff
@@ -0,0 +1,57 @@
+"""
+Copyright (c) 2020 The Orbit Authors. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file.
+"""
+
+import logging
+
+from core.orbit_e2e import Fragment
+from core.e2e_utils import first
+
+
+class MoveFunctionsTab(Fragment):
+    def right_click_move_context(self, item):
+        item.click_input(button='right')
+        context_menu = self.application.window(best_match="TabBarContextMenu")
+        first(context_menu, "MenuItem", name_contains="Move").click_input(button='left')
+        self.test_case.sleep_short()
```

Comment:
Let's try to handle the unavoidable sleeps using this:

https://pywinauto.readthedocs.io/en/latest/code/pywinauto.timings.html

 ```
from pywinauto import timings
...
# somewhere in the initialization, default is 0.09
timings.Timings.after_clickinput_wait= 0.3
```

I filed http://b/174649630 for me to get rid of the sleeps in orbit_testing.py

Reply:
👍